transformations/GermanGenderSwap/test.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I'd like to see more complex test cases here like "Sie ist meine Mutter und er ist mein Vater" or "Er ist ein Artzt und mein Vater".

Reply:
@sebastianGehrmann 
The transformation can handle more complex cases now. I've also used the black  code formatter and added limitations in the ReadMe file, along with adding the names into a json file.